Makefile

```diff
@@ -299,14 +299,16 @@ clean:
 	( cd apps/aesophia/test/contracts && $(MAKE) clean; )
 	( cd $(HTTP_APP) && $(MAKE) clean; )
 	@$(MAKE) multi-distclean
-	@rm -rf _build/system_test+test _build/system_test _build/test _build/prod _build/local
+	@rm -rf eqc-plain eqc-system-test
+	@rm -rf _build/system_test+eqc+test _build/system_test+test+eqc _build/system_test+test _build/system_test _build/test _build/prod _build/local
 	@rm -rf _build/default/plugins
 	@rm -rf $$(ls -d _build/default/lib/* | grep -v '[^_]rocksdb') ## Dependency `rocksdb` takes long to build.
 
 distclean: clean
 	( cd apps/aecuckoo && $(MAKE) distclean; )
 	( cd otp_patches && $(MAKE) distclean; )
 	( cd $(HTTP_APP) && $(MAKE) distclean; )
+	@rm -rf eqc
```

Comment:
No, please not. I keep the eqc directory with all my properties as included repo.... this would throw away a lot of my work!